client/src/components/QuestionsAndAnswers/QAList/QAListItem/AnswerList/AnswerListItem.jsx

```diff
@@ -12,6 +13,13 @@ const AnswerListItem = ({ answer, helpfulAnswerClickHandler, reportButtonClickHa
     month: "short",
     year: "numeric"
   }
+  const helpfulAnswerClickHandler = () => {
+    console.log('clicked helpful on answer: ', answer?.answer_id);
+    //should only be able to click once!
+    axios.put(`/qa/answers/${answer.answer_id}/helpful`)
+      .then( () => {setHelpfulCount(helpfulCount + 1)})
```

Comment:
Do you need to call this setHelpfulCount function here? I believe the put request to the helpful endpoint will automatically add + 1 to its count but I could be mistaken

Reply:
for now i do since i am not rerendering the list when the api call is kicked off, will be faster for me to just reorder the list based off of the id that was changed.